src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -53,15 +56,26 @@ const ImagePreviewContainer = styled.View`
   marginBottom: 10
 `
 
+const PDFPreviewContainer = styled.View`
+  marginBottom: 10
+`
+
 interface Props extends RelayProps {
   senderName: string
   artworkPreview?: JSX.Element
-  imagePreview?: JSX.Element
 }
 
 export class Message extends React.Component<Props, any> {
   render() {
     const date = moment(this.props.message.created_at).fromNow(true)
+    const attachments = this.props.message.attachments
+    let hasImageAttachment
+    let hasPDFAttachment
+    if (attachments.length > 0) {
+      hasImageAttachment = attachments[0].content_type === "image/jpeg"
```

Comment:
> we probably can support a wider array (or maybe just adding png is sufficient?)

Is there a reason to limit at all? Unless it’s an animated GIF, iOS should be able to render most all images. So how about just checking if the first component of the content-type is `image`?